test/system/test_tpm2_pcrevent.sh

```diff
@@ -58,10 +58,18 @@ tpm2_pcrevent -Q < $hash_in_file
 
 # Test that fifo stdin works
 cat $hash_in_file | tpm2_pcrevent > $hash_out_file
+cat $hash_out_file
 
 # Verify output as expected.
 for l in `cat $hash_out_file`; do
   alg=`echo -n $l | cut -d\: -f 1-1`
+  if [ x"$alg" == x"sha" ]; then
+    alg="sha1"
+  elif [ x"$alg" == x"sm3_256" ]; then
+    echo "Ignore checking the hash algorithm $alg"
+    continue
+  fi
```

Comment:
the elif clause is pointless, delete it.

Reply:
Instead of hardcoding the algorithms that don't have a sum command, couldn't you check if "$alg"sum exists?

Reply:
OK thanks I will send a new PR.